src/index.html

```diff
@@ -54,5 +54,11 @@
   </header>
   <div class="container" id="main-content">
   </div>
+
+  <div class="alert alert-danger error-popup alert-dismissible hidden">
```

Comment:
I jsut noticed that your template is in `index.html`

I'd suggest you use some template frameworks to generate it in js like mustashe or jade